files/icons/awesome.toml

```diff
@@ -81,3 +81,5 @@ weather_snow = "\uf2dc" # fa-snowflake-o
 weather_sun = "\uf185" # fa-sun-o
 weather_thunder = "\uf0e7" # fa-bolt
 xrandr = "\uf26c" # fa-television
+cpu_boost_on = "\u23fc"
```

Comment:
So which icon set is this actually from?

Reply:
to be honest, I don't know. I just search symbol in unicode table (https://unicode-table.com/en/23FC/)...